subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/incremental/asm/ClassDependenciesVisitor.java

```diff
@@ -35,33 +38,98 @@
     private final static int API = Opcodes.ASM5;
     private static final MethodVisitor EMPTY_VISITOR = new MethodVisitor(API, null) {
     };
+    private static final Set<String> PRIMITIVES = ImmutableSet.<String>builder()
```

Comment:
byte missing. There's a helper class (`Primitives`) in Guava for this, in case that helps.